test/tools/provisioner-cli/main.go

```diff
@@ -83,11 +88,33 @@ func main() {
 			}
 		}
 
+		log.Info("Deploying kbs")
+		keyBrokerService, err := pv.NewKeyBrokerService(props["CLUSTER_NAME"])
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		if err = keyBrokerService.Deploy(context.TODO(), cfg, props); err != nil {
+			log.Fatal(err)
+		}
+
+		var kbsSvcIP string
+		kbsSvcIP, err = keyBrokerService.GetKbsSvcIP(context.TODO(), cfg)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		kbsparams := "cc_kbc::http:" + kbsSvcIP + ":8080"
+		log.Infof("KBS PARAMS: %q:", kbsparams)
```

Comment:
There is an extra colon hence this looks like this: `INFO[0138] KBS PARAMS: "cc_kbc::http:10.0.241.140:8080":`.

```suggestion
		log.Infof("KBS PARAMS: %s", kbsparams)
```

Also my bad on suggesting `%q` before because it does not look right.